src/Conversion/KrnlToLLVM/KrnlGlobal.cpp

```diff
@@ -319,10 +337,15 @@ class KrnlGlobalOpLowering : public ConvertToLLVMPattern {
 
     int32_t index = 0;
     Value lastValue = array;
-    for (const LLVM::GlobalOp &globalOp : globalOps) {
-      Value strAddr = krnl::getPtrToGlobalString(globalOp, loc, builder);
+    auto baseAddr = create.llvm.addressOf(globalStr);
```

Comment:
Please use `Value` instead of `auto`, we follow LLVM convention that avoids `auto` as much as possible.

Reply:
Fixed.